gateleen-core/src/main/java/org/swisspush/gateleen/core/http/UriBuilder.java

```diff
@@ -0,0 +1,51 @@
+package org.swisspush.gateleen.core.http;
+
+
+import java.util.regex.Pattern;
+
+/**
+ * Helper Class for URI manipulations
+ */
+public class UriBuilder {
+
+    private static Pattern uriCleanPattern = Pattern.compile("(?<!(http:|https:))/+");
```

Comment:
*Constants* in Java are typically 1. `static` 2. `final` and 3. named `UPPER_CASE`
This is wide used convention and we should *make it so*

Reply:
Arghh